.github/pull_request_template.md

```diff
@@ -0,0 +1,39 @@
+<!-- 
+    Thank you for your contribution to the polyPod repo. ♡
+
+    Please, make sure that:
+    - Your code builds clean without any errors or warnings (You've run `./build.js` locally)
+    - Tests for the changes have been added, if possible (for bug fixes / new features)
+    - Docs have been reviewed and added / updated, if needed (for bug fixes / new features)
+    - Commits in this PR are minimal and [have descriptive commit messages.](<https://cbea.ms/git-commit/>) <https://cbea.ms/git-commit/>
```

Comment:
Not really something we've agreed on. But I understand this will be seen also by whoever creates a PR from outside polypoly, so it might help.

Reply:
is "Commits in this PR are minimal and have descriptive commit messages with no single emojis" satisfactory? 😆

Reply:
I'm all for that, really. But not something we will really enforce, I guess. It helps reviews, or course.